packages/mds-stream-processor/processors/vehicle-event-processor.ts

```diff
@@ -62,6 +62,18 @@ const processVehicleEvent: StreamTransform<VehicleEvent, LabeledVehicleEvent> =
       geographyLabeler({ telemetry }),
       latencyLabeler({ timestamp, recorded })
     ])
+    const flattenedTelemetry = telemetry
+      ? flattenTelemetry({ ...telemetry, recorded })
+      : {
+          telemetry_timestamp: null,
+          telemetry_accuracy: null,
+          telemetry_altitude: null,
+          telemetry_charge: null,
+          telemetry_heading: null,
+          telemetry_lat: null,
+          telemetry_lng: null,
+          telemetry_speed: null
+        }
```

Comment:
I feel like the `null` telemetry case should be handled by `flattenTelemetry` ... also, why not just create a `telemetryLabeler` that does this the same way that other labelers work.

Reply:
The issue is that Telemetry can be null in the context of the vehicle-event-processor, but it's never null in the context of the vehicle-telemetry-processor. We need to handle the `null` case somewhere if we type it so that the result of the labeler can be null, but if we do that in the telemetry processor it just feels wrong (considering we'll never have it be null in that context)

Reply:
The result of the labeler won't be null ... it will be
```ts
{
  telemetry_timestamp: null,
  telemetry_accuracy: null,
  telemetry_altitude: null,
  telemetry_charge: null,
  telemetry_heading: null,
  telemetry_lat: null,
  telemetry_lng: null,
  telemetry_speed: null
}
```

Reply:
I meant the data _within_ it will be null. If we do that, we're basically just sending a telemetry record through the pipeline where everything* is nullable